index.html

```diff
@@ -210,15 +245,23 @@ <h1>
                   <button type="button" class="primary-btn">See Project</button>
                 </div>
               </div>
-            </div>
-          </div>
+              </div>
+                </div>
+              </div>
+
+            
+        
         </section>
 
         <!-- Work-section Ends here -->
 
         <!-- About-section Starts here -->
         <section id="about-section">
           <div class="about-title container">
+            <div class="row">
+              <div class="left">
+             
+           
             <h1>
```

Comment:
- [ ] Great job working on your about section! 💯  However, remember that the main goal of this project is that develop an entire website based on a given design. 🎨 Please, consider fixing these section details too. 🤓 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199852272-0e2373de-f2dc-486b-a6de-78bafe9fea5a.png) | ![imageedit_14_7317283779](https://user-images.githubusercontent.com/77069011/199852206-a1bafe3b-f0e7-462c-9c10-5d7a548f6201.gif) |